pandas/formats/format.py

```diff
@@ -1277,29 +1277,27 @@ def _write_hierarchical_rows(self, fmt_values, indent):
 
 
 def _get_level_lengths(levels, sentinel=''):
```

Comment:
While you have rewritten this function a bit, would you like to add a docstring at once as well?